content/en/docs/reference/tools/_index.md

```diff
@@ -29,6 +29,11 @@ to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resou
 [Helm](https://helm.sh/) is a tool for managing packages of pre-configured
 Kubernetes resources. These packages are known as _Helm charts_.
 
+{{% thirdparty-content %}}
```

Comment:
Hi @RA489 .
I think you only need to add the thirdparty-content shortcode once. The preview shows the third party warning message twice.
Can you remove these additions and add the thirdparty-content shortcode at line 15?
```
{{% thirdparty-content %}}
```
Thank you.

Page preview:
https://deploy-preview-27428--kubernetes-io-main-staging.netlify.app/docs/reference/tools/#helm

Reply:
> Hi @RA489 .
> I think you only need to add the thirdparty-content shortcode once. The preview shows the third party warning message twice.
> Can you remove these additions and add the thirdparty-content shortcode at line 15?
> 
> ```
> {{% thirdparty-content %}}
> ```
> 
> Thank you.
> 
> Page preview:
> https://deploy-preview-27428--kubernetes-io-main-staging.netlify.app/docs/reference/tools/#helm

sure.

Reply:
removed.